src/app/components/Block/content.scss

```diff
@@ -212,7 +214,30 @@
     .Block-content[class*='u-richtext'] > * {
       -webkit-backdrop-filter: blur(5px);
       backdrop-filter: blur(5px);
-      background-color: var(--od-colour-theme-surface-over-image);
+      position: relative;
+      overflow: hidden;
+      // This section is a little convoluted.
+      // The reason the background colour is applied as a pseudo element and not
+      // as a simple background colour is that we want to be able to utilise the
+      // global --bg variable to take the tint from the background of the page
+      // Unfortunately, that colour doesn't have any opacity, so we need to add
+      // that. This may become possible with the color() function as browser
+      // support improves.
+      // Adding opacity to the entire element has two down sides (1) transparency
+      // on the text and (2) backdrop-filter() only works with background- styles
+      // which have some transparency.
+      &::before {
+        content: '';
+        position: absolute;
+        z-index: -1;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        background-color: var(--bg, var(--od-colour-theme-surface-over-image));
+        transition: background-color 2s;
+        opacity: 0.75;
+      }
```

Comment:
Is there an equivalent to this for the legacy version?

Reply:
There's on `::before` equivalent in legacy and the element bg colours are already in a where, I think we're good.

Reply:
I tested this on house-sitting, btw and it still works fine, but would obsolete one of your overrides @jtfell